frontend/www/js/omegaup/components/arena/NewClarificationPopup.vue

```diff
@@ -129,7 +129,7 @@ export default class ArenaNewClarificationPopup extends Vue {
     };
     this.$emit('new-clarification', {
       clarification: clarificationRequest,
-      target: this,
+      clearForm: this.clearForm(),
```

Comment:
```suggestion
      clearForm: () => this.clearForm(),
```

si no, se está llamando `clearForm()` en ese lugar y es equivalente a tener `clearForm: undefined`, porque ese es el valor de retorno de `clearForm()`).

Reply:
Listo